update-testsuite.sh

```diff
@@ -117,13 +117,16 @@ for repo in ${repos}; do
     if [ $(git status -s ${wast_dir}/*.wast | wc -l) -ne 0 ]; then
         log_and_run git add ${wast_dir}/*.wast
 
-        repo_sha=$(git -C repos/${repo} log --max-count=1 --pretty=oneline | sed -e 's/ .*//')
-        echo "  ${repo}: ${repo_sha}" >> commit_message
+        repo_sha=$(git -C repos/${repo} log --max-count=1 --oneline origin/master| sed -e 's/ .*//')
```

Comment:
So addition of `origin/master` will give use the real commit, not the merge commit?

Reply:
Yup.